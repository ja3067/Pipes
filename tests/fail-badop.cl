x = 3

if x == 3:
	b = True

print(3)

b + 1

print(5)

