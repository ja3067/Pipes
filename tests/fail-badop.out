3
RuntimeError: unsupported operand type(s) for binary +
